code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -924,6 +968,51 @@
 		icon_state = initial(icon_state)
 		item_state = initial(item_state)
 
+/obj/item/clothing/accessory/dogtags/proc/update_name()
+	name = "[src.name] - [src.dogtags_name]"
+
+/obj/item/clothing/accessory/dogtags/examine(mob/user, distance)
+	..(user, distance)
+	if(distance <= 1)
+		show_tag(user)
+	else
+		to_chat(user, SPAN_NOTICE("You have to go closer if you want to read them."))
```

Comment:
```suggestion
		to_chat(user, SPAN_WARNING("You have to go closer if you want to read them."))
```
Feedback failure, should use warning

Reply:
Fair, but wouldn't `SPAN_NOTICE` be more in-line with other 'approach to read' items like papers and business cards? I find `SPAN_WARNING`'s ominous red text to be too alarming for something like this.

Reply:
notice is fine